docs/framework/data/adonet/ef/language-reference/how-entity-sql-differs-from-transact-sql.md

```diff
@@ -133,27 +133,27 @@ group by b + c
   
  Collection-based aggregates operate on collections and produce the aggregated result. These can appear anywhere in the query, and do not require a `group by` clause. For example:  
   
-```  
+```sql  
 select t.a as a, count({1,2,3}) as b from T as t     
 ```  
   
  [!INCLUDE[esql](../../../../../../includes/esql-md.md)] also supports SQL-style aggregates. For example:  
   
-```  
+```sql  
 select a, sum(t.b) from T as t group by t.a as a  
 ```  
   
 ## ORDER BY Clause Usage  
  Transact-SQL allows ORDER BY clauses to be specified only in the topmost SELECT .. FROM .. WHERE block. In [!INCLUDE[esql](../../../../../../includes/esql-md.md)] you can use a nested ORDER BY expression and it can be placed anywhere in the query, but ordering in a nested query is not preserved.  
```

Comment:
```suggestion
Transact-SQL allows `ORDER BY` clauses to be specified only in the topmost `SELECT .. FROM .. WHERE` block. In [!INCLUDE[esql](../../../../../../includes/esql-md.md)] you can use a nested `ORDER BY` expression and it can be placed anywhere in the query, but ordering in a nested query is not preserved.  
```